identify/extensions.py

```diff
@@ -223,21 +225,26 @@
     '.bash_profile': EXTENSIONS['bash'],
     '.bashrc': EXTENSIONS['bash'],
     '.bowerrc': EXTENSIONS['json'] | {'bowerrc'},
+    '.browserslintrc': {'txt', 'browserslintrc'},
+    '.codespellrc': EXTENSIONS['ini'] | {'codespellrc'},
     '.coveragerc': EXTENSIONS['ini'] | {'coveragerc'},
     '.cshrc': EXTENSIONS['csh'],
+    '.csslintrc': EXTENSIONS['json'] | {'csslintrc'},
     '.dockerignore': {'text', 'dockerignore'},
     '.editorconfig': {'text', 'editorconfig'},
     '.flake8': EXTENSIONS['ini'] | {'flake8'},
     '.gitattributes': {'text', 'gitattributes'},
     '.gitconfig': EXTENSIONS['ini'] | {'gitconfig'},
     '.gitignore': {'text', 'gitignore'},
+    '.gitlint': EXTENSIONS['ini'] | {'gitlint'},
     '.gitmodules': {'text', 'gitmodules'},
     '.hgrc': EXTENSIONS['ini'] | {'hgrc'},
     '.jshintrc': EXTENSIONS['json'] | {'jshintrc'},
     '.mailmap': {'text', 'mailmap'},
     '.mention-bot': EXTENSIONS['json'] | {'mention-bot'},
     '.npmignore': {'text', 'npmignore'},
     '.pdbrc': EXTENSIONS['py'] | {'pdbrc'},
+    '.pylintrc': EXTENSIONS['ini'] | {'pylintrc'},
```

Comment:
I don't see this used in the documentation, the name there is `pylintrc` without a dot

Reply:
Both are listed [here](http://pylint.pycqa.org/en/latest/user_guide/run.html#command-line-options) in several places, and I've seen both used (unfortunately, I can't seem to search for one or the other separately on Github to evaluate actual numbers).